torch/_torch_docs.py

```diff
@@ -5787,6 +5787,90 @@ def merge_dicts(*dicts):
     tensor([  2.,   4.,   8.,  16.])
 """.format(**common_args))
 
+add_docstr(torch.float_power,
+           r"""
+float_power(input, exponent, *, out=None) -> Tensor
+
+Takes the power of each element in :attr:`input` with :attr:`exponent` and
+returns a tensor in float64 with the result.
```

Comment:
"Raises :attr:\`input\` to power of :attr:\`exponent\`, elementwise, in double precision. If neither input is complex returns a  \`\`torch.float64\`\` tensor, and if one or more inputs is complex returns a \`\`torch.complex128\`\` tensor."

You can just use the mathematical expression with the subscript notation immediately after the intro paragraph. No need to clarify the scalar vs. tensor case (the scalar case is implied with broadcasting).

Reply:
Addressed!